yarn.lock

```diff
@@ -6733,6 +6733,23 @@ __metadata:
   languageName: unknown
   linkType: soft
 
+"@backstage/plugin-gateway-backend@workspace:plugins/gateway-backend":
+  version: 0.0.0-use.local
+  resolution: "@backstage/plugin-gateway-backend@workspace:plugins/gateway-backend"
+  dependencies:
+    "@backstage/backend-app-api": "workspace:^"
+    "@backstage/backend-defaults": "workspace:^"
+    "@backstage/backend-plugin-api": "workspace:^"
+    "@backstage/backend-test-utils": "workspace:^"
+    "@backstage/cli": "workspace:^"
+    "@opentelemetry/api": "npm:^1.9.0"
+    "@opentelemetry/core": "npm:^1.30.0"
```

Comment:
can this even be ^1.29.0? That reduces the lockfile change even more

Reply:
you'll need to check out the unchanged master yarn.lock first and then rerun yarn install again, because it's already resolved to the double block of both 1.29.0 and 1.30.1 below